django/test/client.py

```diff
@@ -158,6 +154,10 @@ def __call__(self, environ):
             response.close()                    # will fire request_finished
             request_finished.connect(close_old_connections)
 
+        # Attach the originating request to the response so that it could be
+        # later retrieved.
+        response.wsgi_request = request
```

Comment:
@lajarre yes, it is.

Tests passed. I think this change should not bring any issue.